taskcluster/script.py

```diff
@@ -204,7 +192,7 @@ def main():
 
     # enable charging on device if it is disabled
     #   see https://bugzilla.mozilla.org/show_bug.cgi?id=1565324
-    rc = enable_charging(device, device_type) + rc
+    enable_charging(device, device_type)
```

Comment:
Since we are ignoring the return value of enable_charging, should we go ahead and remove the rc stuff from it altogether?

Reply:
Ah, yeah, I missed the `rc = 0`. Gone now.